api/validation/validators.py

```diff
@@ -33,3 +35,9 @@ def is_alphanumeric_with_hyphen(arg: str):
 
 def valid_api_log_levels_predicate(loglevel):
     return loglevel.lower() in VALID_LOG_LEVELS
+
+def size_not_exceeding(data, size):
```

Comment:
todo: don't forget to add some tests on the new validator and on the `raise_on_missing_body` flag as well